src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/IslHelper.groovy

```diff
@@ -189,7 +190,13 @@ class IslHelper {
     //TOOD: replace boolean parameter with enum FORCE/NOT_FORCE
     def deleteIsl(Isl isl, boolean isForce = false) {
         cleanupManager.addAction(RESTORE_ISL, {
-            northbound.portDown(isl.srcSwitch.dpId, isl.srcPort)
+            def links = northbound.getAllLinks()
+            def forwardIsl = islUtils.getIslInfo(links, isl.reversed)
+            def reverseIsl = islUtils.getIslInfo(links, isl)
```

Comment:
although the way we name forwardIsl and reverseIsl does not affect forthcoming logic, 
but just in case, is it a typo or intentionally?
**forwardIsl** = islUtils.getIslInfo(links, **isl.reversed**)
**reverseIsl** = islUtils.getIslInfo(links, **isl**)
if it is a typo, then better to fix it:)

Reply:
typo